ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v1/fileinput/fileinput.html

```diff
@@ -36,7 +36,7 @@
     <input class="cmp-adaptiveform-fileinput__widget"
            data-sly-attribute.multiple="${file.isMultiple ? 'multiple' : ''}"
            style="${file.buttonText ? 'display:none' : '' @ context='styleString'}"
-           title="${datePicker.tooltipVisible ? '' : datePicker.tooltipText}"
+           title="${file.tooltipVisible ? '' : file.tooltipText}"
```

Comment:
There is a cypress test case for localization, can you add this test there ?

Reply:
Currently this test is a WIP for core-components, 
PR: https://github.com/adobe/aem-core-forms-components/pull/542
I've requested @egmox to include this scenario also.

cc; @rismehta